From 518fe5aea452a4660e07855716349a0c84e59b14 Mon Sep 17 00:00:00 2001
From: Karthik Ramanan <a0393906@ti.com>
Date: Tue, 21 Jan 2014 11:48:19 +0000
Subject: [PATCH] temp-hack: to enable weston 1.3

This patch resolves the issue with the black display seen
when running weston 1.3 version on the dra7xx target.

Known issues:
1. simple-egl application does not work
2. weston on multiple displays does not work

Signed-off-by: Karthik Ramanan <a0393906@ti.com>
---
 src/compositor-drm.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/compositor-drm.c b/src/compositor-drm.c
index 0ac5efa..dd28f0f 100644
--- a/src/compositor-drm.c
+++ b/src/compositor-drm.c
@@ -616,7 +616,7 @@ drm_output_repaint(struct weston_output *output_base,
 			.request.sequence = 1,
 		};
 
-		if ((!s->current && !s->next) ||
+		if ((!s->current && !s->next) &&
 		    !drm_sprite_crtc_supported(output_base, s->possible_crtcs))
 			continue;
 
-- 
1.7.9.5

